### DISCUSSION TITLE
are there any breaking changes on v3?

### DISCUSSION THREAD

**Discussion by ta3pks** [routerify/routerify#109] [Q&A]
I maintain a fork of [WebSocket library](https://docs.rs/routerify-ws) with some minor modifications. And I make use of routerify in production pretty heavily. 
Are there any breaking changes I should be aware of before updating?
According to the changelog there's not really much changes between 2 and 3

**Comment by rousan** [comment]
There are no feature specific changes in routerify 3. The version 3 is released because we had to update/minimize the dependencies to the latest versions.

**Reply by ta3pks** [reply]
Seems things are broken at version 3.
I started to get `Routerify: No remote address added while processing request',` after the update

**Reply by rousan** [reply]
@NikosEfthias, Can you please reproduce the same with a simple example code, so that we can figure out what went wrong?
